Protocol/ErrorMessages.xml

```diff
@@ -9090,6 +9090,28 @@
 							</ErrorMessage>
 						</ErrorMessages>
 					</Check>
+					<Check id="77">
+						<Name namespace="Protocol.Params.Param.ArrayOptions">CheckVolatileTableConsistency</Name>
```

Comment:
CheckVolatileTableConsistency is a very specific name. I would rebrand it to CheckVolatileTables.
This way, we can extend it with any results related to volatile table, being regarding consistency or not.